src/modules/wallet/ticket/Ticket.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <div class="flex bg-primary-1 rounded-2xl">

```

Comment:
I initially think this too, but then the number does not really match that well (can not find a good denominator). I can do it in the expense of output not follow design.

Reply:
You can take a look at this. https://stackoverflow.com/questions/46589611/how-to-use-a-css-grid-with-percentages-and-an-unknown-amount-of-columns
The reason I recommend using one grid class instead of multiple flex class because fewer classes improve web performance.